worlds/noita/Items.py

```diff
@@ -0,0 +1,63 @@
+from typing import Dict
+from BaseClasses import Item
+
+class NoitaItem(Item):
+    game: str = "Noita"
+
+# 110000 - 110013
+item_table: Dict[str, int] = {
+    "Bad":              110000,
```

Comment:
This should be called "Trap" for consistency. When other games see it it'll appear as "DaftBrit's Bad".